src/pipecat/services/gladia/stt.py

```diff
@@ -278,6 +279,9 @@ def __init__(
         self._connection_task = None
         self._should_reconnect = True
 
+    def __str__(self):
+        return f"GladiaSTTService [{self._session_id}]"
```

Comment:
In all our services we log `self.name` to include the specific instance, so it would be better to keep the same pattern and just include the session id, like this:
```suggestion
        return f"{self.name} [{self._session_id}]"
```